pkg/nsxt/config/config_ini_legacy.go

```diff
@@ -0,0 +1,40 @@
+package config
+
+import (
+	"fmt"
+
+	"k8s.io/klog"
+)
+
+// ValidateConfig checks NSXT configurations
+func (cfg *NsxtConfigINI) ValidateConfig() error {
+	if cfg.VMCAccessToken != "" {
+		if cfg.VMCAuthHost == "" {
+			msg := "vmc auth host must be provided if auth token is provided"
+			klog.Errorf(msg)
+			return fmt.Errorf(msg)
+		}
+	} else if cfg.User != "" {
+		if cfg.Password == "" {
+			msg := "password is empty"
+			klog.Errorf(msg)
+			return fmt.Errorf(msg)
+		}
+	} else if cfg.ClientAuthCertFile != "" {
```

Comment:
I added certificate check in ValidateConfig(). If client cert and key are provided, we don't need token or password for NSXT connection.
Let me know if it is acceptable.

Reply:
This seems fine, but can we add this in a separate commit after the refactor commit?